src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -67,1008 +50,333 @@ public class ADDR_Addresses_TDTM extends TDTM_Runnable {
-    * @return void
-    */ 
-    public static void createNonDuplicateAddresses(list<Address__c> listAddr, DmlWrapper dmlWrapper, boolean includeAddressType) {  
-        
-        map<Address__c, Address__c> mapAddrDupAddr= findDuplicateAddresses(listAddr);
-        for (Address__c addrNew : listAddr) {
-            Address__c addrDup = mapAddrDupAddr.get(addrNew);
-            if (addrDup != null) {
-                // if it is a fuzzy match, not exact, then we want to update the existing address.
-                if (isAddressChanged(addrNew, addrDup, includeAddressType)) {
-                    copyAddressObj(addrNew, addrDup);
-                    if (addrNew.Default_Address__c)
-                        addrDup.Default_Address__c = true;
-                    dmlWrapper.objectsToUpdate.add(addrDup);
-                } else if (addrNew.Default_Address__c && !addrDup.Default_Address__c) {
-                    addrDup.Default_Address__c = true;
-                    dmlWrapper.objectsToUpdate.add(addrDup);
-               }
-            } else {
-                dmlWrapper.objectsToInsert.add(addrNew);
-            }       
-        }   
-    }
-    
-    /*******************************************************************************************************
-    * @description for the given addresses, find if they have a match, and return a map that
-    * has null for no match, and the matching address if there is a match.
-    * @param listAddr a list of Addresses to update
-    * @return map<Address__c, Address__c>
-    */ 
-    public static map<Address__c, Address__c> findDuplicateAddresses(list<Address__c> listAddr) {      
-        // for each address, create a map of HHId to lists of Addresses
-        list<Id> listAccId = new list<Id>();
-        for (Address__c addr : listAddr)
-           listAccId.add(addr.Parent_Account__c);
-        map<Id, list<Address__c>> mapAccIdListAddrHH = getMapAccIdListAddrHH(listAccId);
-        boolean tryFuzzy = UTIL_CustomSettingsFacade.getSettings().Simple_Address_Change_Treated_as_Update__c;
-        
-        // create our map for returning information in
-        map<Address__c, Address__c> mapAddrDupAddr = new map<Address__c, Address__c>();
-        
-        // now compare our Addresses to the existing Addresses
-        for (Address__c addr : listAddr) {
-            list<Address__c> listAddrHH = mapAccIdListAddrHH.get(addr.Parent_Account__c);
-            boolean hasMatch = false;
-            if (listAddrHH != null) {
-                Address__c addrMatch = null;
-                for (Address__c addrHH : listAddrHH) {
-                	// only include address type in the test if the new one isn't empty.
-                    if (!isAddressChanged(addr, addrHH, addr.Address_Type__c != null)) {
-                        // found exact match, record it and bail out
-                        addrMatch = addrHH;
-                        break;
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.